core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -245,20 +284,132 @@ public Company getCompanyContext()
+	 */
+	private Actor createNewEmployeeComponent(Employee newRandomEmployee)
+	{
+		Table newEmployeeComponent = new Table();
+		
+		/** Left side of New Employee Component */
+		VerticalGroup newEmployeeInformation = new VerticalGroup();
+		Label name = new Label("Name: " + newRandomEmployee.getName(), skin);
+		Label position = new Label("Position: " + newRandomEmployee.getPosition(), skin);
+		Label preferredHourlyWage = new Label("Preferred Hourly Wage: $"
+				+ Math.round(newRandomEmployee.getPreferredHourlyWage()), skin);
+		newEmployeeInformation.addActor(name);
+		newEmployeeInformation.addActor(position);
+		newEmployeeInformation.addActor(preferredHourlyWage);
+		
+		/** Right side of New Employee Component */
+		VerticalGroup newEmployeeInputs = new VerticalGroup();
+		Label morale = new Label("Morale: "
+				+ Math.round(newRandomEmployee.getMorale() * 100) + "%", skin);
+		Actor pay = createNewEmployeePay(newRandomEmployee, morale);
+		TextButton hire = new TextButton("Hire", skin);
+		hire.addListener(new HireEmployeeListener(newRandomEmployee));
+		newEmployeeInputs.addActor(pay);
+		newEmployeeInputs.addActor(morale);
+		newEmployeeInputs.addActor(hire);
+		
+		newEmployeeComponent.add(newEmployeeInformation).expand().fill();
+		newEmployeeComponent.add(newEmployeeInputs).expand().fill();
+		
+		return newEmployeeComponent;
+	}
+	
+	/**
+	 * The Employee Pay Field Listener will ensure that all fields that are affected by
+	 * the employee's actual wage will be updated when the actual wage is altered.
+	 */
+	private class EmployeePayFieldListener implements TextField.TextFieldListener
+	{
+		private Employee newRandomEmployee;
+		private TextField employeePayField;
+		private Label morale;
+		
+		public EmployeePayFieldListener(Employee newRandomEmployee,
+				TextField employeePayField, Label morale)
+		{
+			this.newRandomEmployee = newRandomEmployee;
+			this.employeePayField = employeePayField;
+			this.morale = morale;
+		}
+		
+		@Override
+		public void keyTyped(TextField textField, char c)
+		{
+			String proposal = employeePayField.getText().trim();
+			int proposalValue = newRandomEmployee.getActualHourlyWage();
+			
+			if (proposal.length() <= 0)
+			{
+				proposalValue = 0;
+			}
+			
+			try
+			{
+				proposalValue = Integer.parseInt(proposal);
+				newRandomEmployee.setActualHourlyWage(proposalValue);
+				morale.setText("Morale: " + newRandomEmployee.getMorale() * 100 + " %");
+			}
+			catch (NumberFormatException exception)
+			{
+				// DO NOTHING
+			}
+			
+			String text = Integer.toString(proposalValue);
+			employeePayField.setText(text);
+			employeePayField.setCursorPosition(text.length());
+			
+		}
+		
+	}
+	
+	/**
+	 * @param newRandomEmployee
+	 * @param morale
+	 * @return Pay field that allows the user to set the potential new employee's actual
+	 *         pay prior to hiring.
+	 */
+	private Actor createNewEmployeePay(Employee newRandomEmployee, Label morale)
```

Comment:
In order for morale to update automatically when a new actual hourly wage is defined by the user. Will work on refactoring the other employee pay field for the model shown when clicking on hired employees so that morale is updated automatically there as well.